### PR TITLE
[step1][Improve]Use flink’s checkstyle to format the source connector

### PR DESCRIPTION
**Pull request by DongLiang-0**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:
The original doris-flink-connector code has not been formatted.
Taking advantage of the opportunity of Contribute Flink Doris Connector to the Flink community, this project formatted using flink’s checkstyle file to try to meet the code style requirements of the flink community.
[Apache Flink Code Style and Quality Guide](https://flink.apache.org/how-to-contribute/code-style-and-quality-preamble/)

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by JNSimba** [comment]
Thank you @DongLiang-0  , this is a good job

File: flink-doris-connector/src/main/java/org/apache/doris/flink/rest/models/BackendRow.java [hunk 1]
```diff
@@ -25,44 +25,49 @@
...
-    private String IP;
+    private String ip;
 
     @JsonProperty(value = "Alive")
-    private Boolean Alive;
+    private Boolean alive;
 
     public String getHttpPort() {
-        return HttpPort;
+        return httpPort;
```
Comment (JNSimba) [inline flink-doris-connector/src/main/java/org/apache/doris/flink/rest/models/BackendRow.java:37]: Will there be a problem if change it to lowercase here?

File: flink-doris-connector/src/main/java/org/apache/doris/flink/rest/models/QueryPlan.java [hunk 2]
```diff
@@ -17,12 +17,19 @@
...
 import java.util.Map;
 import java.util.Objects;
 
 public class QueryPlan {
+    @JsonProperty(value = "status")
     private int status;
-    private String opaqued_query_plan;
+
+    @JsonProperty(value = "opaqued_query_plan")
+    private String opaquedQueryPlan;
```
Comment (JNSimba) [inline flink-doris-connector/src/main/java/org/apache/doris/flink/rest/models/QueryPlan.java:30]: the same as above

File: flink-doris-connector/src/main/java/org/apache/doris/flink/serialization/RowBatch.java [hunk 3]
```diff
@@ -90,10 +89,12 @@ public void put(Object o) {
     private List<FieldVector> fieldVectors;
     private RootAllocator rootAllocator;
     private final Schema schema;
-    private final String DATETIME_PATTERN = "yyyy-MM-dd HH:mm:ss";
-    private final String DATETIMEV2_PATTERN = "yyyy-MM-dd HH:mm:ss.SSSSSS";
-    private final DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern(DATETIME_PATTERN);
-    private final DateTimeFormatter dateTimeV2Formatter = DateTimeFormatter.ofPattern(DATETIMEV2_PATTERN);
+    private final String datetimePattern = "yyyy-MM-dd HH:mm:ss";
+    private final String datetimeV2Pattern = "yyyy-MM-dd HH:mm:ss.SSSSSS";
```
Comment (JNSimba) [inline flink-doris-connector/src/main/java/org/apache/doris/flink/serialization/RowBatch.java:93]: Change to uppercase,   can change it to static

File: flink-doris-connector/src/main/java/org/apache/doris/flink/rest/models/BackendRow.java (hunk 1 above)
Comment (DongLiang-0) [reply flink-doris-connector/src/main/java/org/apache/doris/flink/rest/models/BackendRow.java:37]: The main function here is to map the String type to a Java object through Json. After declaring the mapped field type that needs to be converted through @JsonProperty(value = "xxx"), there is no problem here.

File: flink-doris-connector/src/test/java/org/apache/doris/flink/utils/DateToStringConverter.java [hunk 4]
```diff
@@ -43,23 +43,27 @@ public class DateToStringConverter implements CustomConverter<SchemaBuilder, Rel
     private DateTimeFormatter timestampFormatter = DateTimeFormatter.ISO_DATE_TIME;
     private ZoneId timestampZoneId = ZoneId.systemDefault();
 
-    public static Properties DEFAULT_PROPS = new Properties();
+    public static Properties defaultProps = new Properties();
```
Comment (JNSimba) [inline flink-doris-connector/src/test/java/org/apache/doris/flink/utils/DateToStringConverter.java:46]: Does it have to be static final to be capitalized?

File: flink-doris-connector/src/main/java/org/apache/doris/flink/serialization/RowBatch.java (hunk 3 above)
Comment (DongLiang-0) [reply flink-doris-connector/src/main/java/org/apache/doris/flink/serialization/RowBatch.java]: Yes, changed!

File: flink-doris-connector/src/test/java/org/apache/doris/flink/utils/DateToStringConverter.java (hunk 4 above)
Comment (DongLiang-0) [reply flink-doris-connector/src/test/java/org/apache/doris/flink/utils/DateToStringConverter.java]: Yes, changed

**Review by JNSimba — APPROVED** [review]
LGTM

**Pull request merged**